### ISSUE TITLE
Added support for .yml configs broke auto-generation of uuid,name,manager in service configs

### ISSUE THREAD

**Issue by stianstr** [ChainsAutomation/chains#23]
When .yml config support was added, config management in Orchestrator was changed to be "config parser agnostic". What was overlooked was that the config parser was also used to _write_ auto generated fields to new configs (uuid, name, manager).

That code no longer works, which means that when a new service config without any of these fields, orchestrator will crash.

Workaround:
Ensure uuid, name, manager are added manually for all new service configs you add.

Suggested fix:
- Add write-to-config support for .yml parser
- Fix the code that writes to configs in orchestrator

**Comment by olekenneth** [comment]
I'm unable to list any not started services after updating to version with yml-config. @chrisaq do you see all your services when running `chains service list`?

**Comment by chrisaq** [comment]
@olekenneth yes:

```
root@puppeteer:/srv/chains/misc/webgui# chains service list

------------------------------------------------------------------------------------------
Service         Online   Manager         Heartbeat       ID
------------------------------------------------------------------------------------------
philipshue      Online   chainsmaster    0 sec ago       30b98476df614c67802f830a1bf16798
timer           Online   chainsmaster    0 sec ago       1c3bdb0ed7fc475fab1c1d4298a0545a
tellstick       Online   chainsmaster    0 sec ago       14ea92d1350b4d969863c962b086ec11
system          Online   chainsmaster    0 sec ago       1d32b7fa2b9144bb8d52e5a434cebf33
proximitynet    Online   chainsmaster    0 sec ago       d7021fd76322453f8a7212196a38d342
None            Online                   0 sec ago       None
system-mediacenter          mediacenter     about 1 hours ago b7bd68e90d5d44c3a785fc2f10c40ab4
influx          Online   chainsmaster    0 sec ago       3967176ae8a94d7f92a3fe8b646ed0d9
integra                  mediacenter     about 1 hours ago 2002c1d695c34c8c8d560845a6df0da5
usbinfo         Online   chainsmaster    0 sec ago       dde1ef24ef674ec183daad5e578b4127
system-rpi-z    Online   rpi-z           0 sec ago       130345c1a89143ac9b130e8e6bab91ef
```

**Comment by chrisaq** [comment]
Well in fact I do not see the lifx-service, here named None.
However that service just doesn't work, so that might be the reason.

**Comment by stianstr** [comment]
Fixed as of c212cb644fb3a33f4100da6134c96910993fe57b
